ci.py

```diff
@@ -405,7 +414,7 @@ def update_docker_compose(modules, pipeline):
     compose_dict['version'] = '2'
 
     try:
-        with open(CI_COMPOSE_FILE, 'w') as docker_compose:
+        with open(CI_COMPOSE_FILE, 'wb') as docker_compose:
```

Comment:
why do these need the `b` flag?

Reply:
the b flag makes it Windows compatible, and having it or not having it has no influence on Linux versions, so it isn't needed at all.